src/python/pants/backend/python/rules/pytest_coverage.py

```diff
@@ -106,41 +114,32 @@
     )
 )
 
-DEFAULT_COVERAGE_CONFIG = dedent(
```

Comment:
This, `get_coveragerc_input()`, and`ensure_section()` right below, were inlined.